ibmcloud-powervs/image/Makefile

```diff
@@ -0,0 +1,22 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+include ../../podvm/Makefile.inc
+
```

Comment:
You can reduce changes in Make.inc and .gitignore by overriding `SKOPEO_BIN` here like this.

```
SKOPEO_BIN = /usr/bin/skopeo
```

Reply:
Thanks for the suggestion @yoheiueda. I made the changes, PTAL!